packages/gasket-plugin-manifest/lib/configure.js

```diff
@@ -0,0 +1,20 @@
+const deepmerge = require('deepmerge');
+const baseConfig = require('./base-config');
+const { prepareStaticOutputPath } = require('./utils');
+
+/**
+ * Configure lifecycle to set up manifest with defaults
+ *
+ * @param {Gasket} gasket - Gasket
+ * @param {Object} config - Base gasket config
+ * @returns {Object} config
+ */
+module.exports = function configure(gasket, config = {}) {
+  const { config: { root } } = gasket;
+  const manifest = deepmerge(baseConfig, config.manifest);
+  const { staticOutput = false } = manifest;
+
+  manifest.staticOutput = prepareStaticOutputPath(staticOutput, root);
```

Comment:
This `prepareStaticOutputPath` code is very simple and only used in this configure hook. Can you simply this by just inlining it here instead of extracting it to a separate util function?